app/views/users/index.html.erb

```diff
@@ -31,15 +32,18 @@
           <td><%= user.rate&.code %></td>
           <td><%= user.dedication %></td>
         <% end %>
+        <td>
+          <%= user.active ? 'Active' : 'Inactive' %>
+        </td>
         <td>
           <%= link_to 'Show', user %>
           <% if can? :edit, user %>
             |
             <%= link_to('Edit', edit_user_path(user)) %>
           <% end %>
-          <% if can? :delete, user %>
+          <% if user.active && (can? :delete, user) %>
             |
-            <%= link_to('Destroy', user, method: :delete, data: { confirm: 'Are you sure?' }) %>
+            <%= link_to('Terminate', user, method: :delete, data: { confirm: 'Are you sure?' }) %>
```

Comment:
Maybe instead of "Terminate" we could just call it "Archive" or "Make inactive"? Terminate sounds quite powerful and nice, but maybe too strong =D But it doesn't hurt if it stays like this.

Reply:
Yeah, I liked terminate because of that :-) but I'll change it